src/integrations/Clevertap/browser.js

```diff
@@ -48,19 +48,32 @@ class Clevertap {
 
   init() {
     logger.debug('===in init Clevertap===');
+    // START-NO-SONAR-SCAN
     const sourceUrl =
-      document.location.protocol == 'https:'
-        ? 'https://d2r1yp2w7bby2u.cloudfront.net/js/a.js'
-        : 'http://static.clevertap.com/js/a.js';
+      document.location.protocol === 'https:'
+        ? 'https://d2r1yp2w7bby2u.cloudfront.net/js/clevertap.min.js'
+        : 'http://static.clevertap.com/js/clevertap.min.js';
+    // END-NO-SONAR-SCAN
```

Comment:
Why is the scanning turned off for just one file?

Reply:
Sonar was not allowing http endpoint of clevertap so turned off scanning for the urls.

Reply:
Got it. You might want to add an inline comment to clarify that.